tests/pages/Home.test.js

```diff
@@ -28,13 +29,13 @@ describe('Home', ()=> {
   });
 
   it('render without crash', ()=> {
-    const wrapper = shallow(<Home rootStore={rootStore} />);
+    const wrapper = shallow(<Provider rootStore={rootStore}><Home /></Provider>);
```

Comment:
i think `Provider`should not be imported, since component testing should be as pure as possible, not affected by additional component.

Reply:
``Provider`` just provide ``rootStore`` props to child component, no more side effects. btw,``Provider`` working the same in the real code

Reply:
https://github.com/mobxjs/mobx-react/issues/260, however we need  work a better way to fix container component test.

Reply:
ok